hopp_tools_steel.py

```diff
@@ -2199,11 +2306,11 @@ def hydrogen_storage_capacity_cost_calcs(H2_Results,electrolyzer_size_mw,storage
     
     # Get average electrolyzer efficiency
     electrolyzer_efficiency_while_running = []
-    for j in range(len(H2_Results['electrolyzer_total_efficiency'])):
+    for j in range(len(H2_Results['water_hourly_usage'])):
         if H2_Results['hydrogen_hourly_production'][j] > 0:
             electrolyzer_efficiency_while_running.append(H2_Results['electrolyzer_total_efficiency'][j])
```

Comment:
This looks like an incomplete loop/if statement. Also not sure why the water hourly usage is being used to loop over the hydrogen hourly production. Loops should be over the relevant iteratable. We need to make sure everything runs before merging. This should fail as is.

Reply:
I agree. It seems to work without issue as it is here so I suggest we leave it for now, but note that in the future we should replace it with a more accurate calculation of average efficiency as you noted in point # 2 above.